plugins/scaffolder/src/components/fields/RepoUrlPicker/validation.ts

```diff
@@ -48,13 +48,19 @@ export const repoPickerValidation = (
           );
         }
 
+        if (!searchParams.get('project')) {
+          validation.addError(
+            'Incomplete repository location provided, project not provided',
+          );
+        }
+      } else if (integrationApi?.byHost(host)?.type === 'azure') {
```

Comment:
I could have added the condition to the bitbucket check, but I prefer this

Reply:
Can you add some tests in this `validation.test.ts` file for `azure` in this case :pray:

Reply:
Done.
Please take a look at the tests I wrote and let me know if you want me to add/change anything.